# 
# Submitted:
#
# Benedikt Spranger, 2004-01-27
#
# Error: 
#
# i386-unknown-linux-gnu-gcc -DHAVE_CONFIG_H -I. -I. -I.. -I../stage1
# -I/tmp/ptxdist-bmwm-rsc/i386-unknown-linux-gnu/include -Wall
# -Wmissing-prototypes -Wunused -Wshadow -Wpointer-arith -falign-jumps=1
# -falign-loops=1 -falign-functions=1 -Wundef -O2 -I../lib -DGRUB_UTIL=1
# -DFSYS_EXT2FS=1 -DFSYS_FAT=1 -DFSYS_FFS=1 -DFSYS_JFS=1 -DFSYS_MINIX=1
# -DFSYS_REISERFS=1 -DFSYS_VSTAFS=1 -DFSYS_XFS=1 -DUSE_MD5_PASSWORDS=1
# -DSUPPORT_SERIAL=1 -DSUPPORT_HERCULES=1 -fwritable-strings -g -c -o
# libgrub_a-fsys_reiserfs.o `test -f fsys_reiserfs.c || echo
# './'`fsys_reiserfs.c
# fsys_reiserfs.c:115: error: long, short, signed or unsigned used invalidly for `j_mount_id'
# 
# Description: 
#
# "__u32 long" is bogus...
#
# State:
# 
# unknown
#

diff -urN grub-0.93/stage2/fsys_reiserfs.c grub-0.93-ptx1/stage2/fsys_reiserfs.c
--- grub-0.93/stage2/fsys_reiserfs.c	2002-11-29 18:46:01.000000000 +0100
+++ grub-0.93-ptx1/stage2/fsys_reiserfs.c	2004-01-27 19:10:31.000000000 +0100
@@ -112,7 +112,7 @@
   /* offset in the log of where to start replay after a crash */
   __u32 j_first_unflushed_offset;
   /* mount id to detect very old transactions */
-  __u32 long j_mount_id;
+  __u32 j_mount_id;
 };
 
 /* magic string to find desc blocks in the journal */
